.circleci/config.yml

```diff
@@ -47,7 +47,7 @@ jobs:
          name: End to end tests
          command: make kind
       - store_artifacts:
-          path: /tmp/kind-logs
+          path: /tmp/etcd-e2e
```

Comment:
This is renamed, because I am now collecting cluster-info dump logs, as well as kind logs.